src/libraries/filcrypto/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+menuTitle: Filcrypto
+title: Cryptographic libraries used in Filecoin
+entries:
+- filproofs
+---
+
+{{< incTocMap "/docs/libraries/filcrypto" >}}
```

Comment:
have you checked that this works @porcuquine ?

Reply:
It did for me.